privacy_evaluator/models/torch/fc_neural_net.py

```diff
@@ -0,0 +1,75 @@
+import torch
+from torch import nn
+from typing import Tuple, Optional
+
+
+def FCNeuralNet(
```

Comment:
Please also specify here that this is a model that can be used for testing the framework with data in the CIFAR or MNIST shape.

Reply:
In both `tf/fc_neural_net.py` and `torch/fc_neural_net.py` we have added "Note" in the second line of the docstring to explain it is just aimed at fetching a model for developers' test when a target model is required. Therefore, this is only compatible with the two shapes.